src/celeritas/optical/WavelengthShiftData.hh

```diff
@@ -0,0 +1,80 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/optical/WavelengthShiftData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/data/Collection.hh"
+#include "celeritas/Types.hh"
+#include "celeritas/grid/GenericGridData.hh"
+
+namespace celeritas
+{
+namespace optical
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Material dependent scalar property of wavelength shift (WLS).
+ */
+struct ScalarPropertyWLS
```

Comment:
Maybe rename `WlsMaterialRecord`? Our style is to try to avoid consecutive blocks of upper-case letters, and this is all the data needed for a single optical material.

The `GenericGridRecord` could be a scalar data member of this class as well, since it's tied to the same optical material.

Reply:
Renamed as suggested.